Makefile

```diff
@@ -121,7 +111,7 @@ SUBLINTERS = deadcode \
 
 # TODO: ^ gochecknoglobals
 
-LINT = $(GOBIN)/golangci-lint run ./... --disable-all --enable 
+LINT = $(GOBIN)/golangci-lint run ./... --disable-all --deadline=5m --enable 
```

Comment:
`LINT` variable seems used when running specific linter for example `$(LINT) structcheck` and looks like 5m deadline is too big for that. 1m or even 30s?

Reply:
cc @akhilkumarpilli for rationale here.